frontend/src/components/Secrets/GSecretDialogDDNS.vue

```diff
@@ -0,0 +1,245 @@
+<!--
+SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+ -->
+
+<template>
+  <g-secret-dialog
+    v-model="visible"
+    :data="secretData"
+    :secret-validations="v$"
+    :secret="secret"
+    vendor="netlify-dns"
+    create-title="Add new DDNS (RFC2136) Secret"
+    replace-title="Replace DDNS (RFC2136) Secret"
```

Comment:
I would remove the `(RFC2136)` from all titles and only Use DDNS or DynDNS Secret

Reply:
@MartinWeindel WDYT?

Reply:
I have no clear preference. The PowerDNS documentation speaks of "Dynamic DNS Update (RFC 2136)". Wikipedia about "Dynamic DNS (DDNS)", but it also says "The second kind of dynamic DNS permits lightweight and immediate updates often using an update client, which do not use the [RFC](https://en.wikipedia.org/wiki/RFC_(identifier)) [2136](https://datatracker.ietf.org/doc/html/rfc2136) standard for updating DNS records". 
Referencing RFC 2136 makes it unambiguous and it is also the provider type name.